galata/test/jupyterlab/zoom.test.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, test } from '@jupyterlab/galata';
+
+test.describe('Low Vision / Zoom Support', () => {
+  // test('Launch Screen at 400% zoom', async ({ page }) => {
+  //   await page.evaluate('document.body.style.zoom = 4.0');
+  //   expect(await page.screenshot()).toMatchSnapshot(
+  //     'launch-screen-at-400-zoom.png'
+  //   );
+  // });
+  const { firefox } = require('playwright');
```

Comment:
Why Firefox?

Reply:
This was meant to be chrome. Sorry my mistake.